src/main/groovy/liquibase/harness/BaseHarnessSuite.groovy

```diff
@@ -1,11 +1,12 @@
 package liquibase.harness
 
 import liquibase.harness.change.ChangeObjectTests
+import liquibase.harness.data.ChangeDataTest
 import liquibase.harness.snapshot.SnapshotObjectTests;
 import org.junit.runner.RunWith;
 import org.junit.runners.Suite;
 
 @RunWith(Suite.class)
-@Suite.SuiteClasses([ChangeObjectTests, SnapshotObjectTests])
+@Suite.SuiteClasses([ChangeDataTest, ChangeObjectTests, SnapshotObjectTests])
```

Comment:
nit
Other our test classes end with [s] what indicates it's not one test but actually number of them based on input